StructureDefinition/NHSDigital-Claim.StructureDefinition.xml

```diff
@@ -317,24 +317,14 @@
     </element>
     <element id="Claim.insurance">
       <path value="Claim.insurance" />
-      <definition value="For EPS this is the NHS Business Services Authority. The sequence value must always be 1.&#xD;&#xA;&#xD;&#xA;```json&#xD;&#xA;&#xD;&#xA;&quot;insurance&quot;:  [&#xD;&#xA;    {&#xD;&#xA;        &quot;sequence&quot;: 1,&#xD;&#xA;        &quot;focal&quot;: true,&#xD;&#xA;        &quot;coverage&quot;: {&#xD;&#xA;            &quot;identifier&quot;: {&#xD;&#xA;                &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/ods-organization-code&quot;,&#xD;&#xA;                &quot;value&quot;: &quot;T1450&quot;&#xD;&#xA;            },&#xD;&#xA;            &quot;display&quot;: &quot;NHS BUSINESS SERVICES AUTHORITY&quot;&#xD;&#xA;        }&#xD;&#xA;    }&#xD;&#xA;],&#xD;&#xA;```" />
+      <definition value="For EPS this is the NHS Business Services Authority (for England) or NWSSP (for Wales). The sequence value must always be 1." />
       <max value="1" />
       <mustSupport value="true" />
     </element>
     <element id="Claim.insurance.coverage.identifier">
       <path value="Claim.insurance.coverage.identifier" />
       <min value="1" />
     </element>
-    <element id="Claim.insurance.coverage.identifier.system">
-      <path value="Claim.insurance.coverage.identifier.system" />
-      <min value="1" />
-      <fixedUri value="https://fhir.nhs.uk/Id/ods-organization-code" />
```

Comment:
Not sure about removal of the fixed ODS code, as this would potentially allow other organization types not listed in ODS to be supplied (though I don't think it would cause a problem for EPS at the moment as they likely have additional validation on top to ensure the code is one supported)

Reply:
@omarisgreat This ask comes about because they want Wales to be able to send Claims to EPS too. However, Wales doesn't use BSA as their reimbursement authority, they use another one. So forcing them to use the ODS code for BSA would be incorrect. However, they agreed with themselves that they can handle the validation at the API side.